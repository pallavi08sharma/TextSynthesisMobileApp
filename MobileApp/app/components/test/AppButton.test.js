import React from 'react';
import renderer from 'react-test-renderer';

import AppButton from '../AppButton'

describe('<AppButton />', () => {
    it('renders correctly', async () => {
      const tree = renderer.create(<AppButton />).toJSON();
      expect(tree).toMatchSnapshot();
    });
  });