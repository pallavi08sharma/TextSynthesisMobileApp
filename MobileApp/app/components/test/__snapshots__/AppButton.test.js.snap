// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`<AppButton /> renders correctly 1`] = `
<View
  accessible={true}
  focusable={false}
  onClick={[Function]}
  onResponderGrant={[Function]}
  onResponderMove={[Function]}
  onResponderRelease={[Function]}
  onResponderTerminate={[Function]}
  onResponderTerminationRequest={[Function]}
  onStartShouldSetResponder={[Function]}
  style={
    Object {
      "alignItems": "center",
      "backgroundColor": "deepskyblue",
      "borderRadius": 25,
      "justifyContent": "center",
      "marginVertical": 10,
      "opacity": 1,
      "padding": 10,
      "width": "100%",
    }
  }
>
  <Text
    style={
      Object {
        "color": "#fff",
        "fontSize": 15,
        "fontWeight": "bold",
        "textTransform": "uppercase",
      }
    }
  />
</View>
`;
